CoreScriptsRoot/Modules/VR/VRHub.lua

```diff
@@ -2,13 +2,64 @@
 --Handles all global VR state that isn't built into a specific module.
 --Written by 0xBAADF00D (Kyle) on 6/10/16
 local CoreGui = game:GetService("CoreGui")
+local UserInputService = game:GetService("UserInputService")
+local RunService = game:GetService("RunService")
+local HttpService = game:GetService("HttpService")
 local RobloxGui = CoreGui:WaitForChild("RobloxGui")
 local Util = require(RobloxGui.Modules.Settings.Utility)
 
+local LaserPointer = require(RobloxGui.Modules.VR.LaserPointer)
+local useLaserPointerSuccess, useLaserPointerValue = pcall(function() settings():GetFFlag("UseLaserPointerVR") end)
+local useLaserPointerFlag = useLaserPointerSuccess and useLaserPointerValue
+
 local VRHub = {}
 local RegisteredModules = {}
 local OpenModules = {}
 
+--VR Setup
+local vrUpdateRenderstepName = HttpService:GenerateGUID(true)
+
+VRHub.LaserPointer = nil
+
+local start, dt = tick(), 1/60
+local function onRenderSteppedLast()
+	local now = tick()
+	dt = now - start
```

Comment:
Could we just initialize dt here instead of outside this func? It looks like it isn't really being used anywhere else.